File: 2016/5.go
package main

import (
	"crypto/md5"
	"encoding/hex"
	"fmt"
)

func main() {

	input := "wtnhxymk"
	password := ""
	index := 0

	for len(password) < 8 {
		id := input + fmt.Sprint(index)
		hash := GetMd5Hash(id)

		if hash[:5] == "00000" {
			password += string(hash[5])
		}

		index++
		fmt.Println(index, password)
	}
}

func GetMd5Hash(text string) string {
	hash := md5.Sum([]byte(text))
	return hex.EncodeToString(hash[:])
}

File: 2016/5.py
import hashlib as h
# file = open('5.txt', 'r')
# input = file.read()
# lines = input.split("\n")



def main ():
    input : str = "abc"
    password : str = ""
    index : int = 3231928

    while len(password) < 8 :

        id = input + str(index)
        hash = h.md5(id.encode()).hexdigest()

        if str(hash).startswith("00000") :
            password += str(hash)[5]

        index += 1
        print(index, password)

    print(password)

main()